src/scene.ts

```diff
@@ -1504,6 +1505,17 @@ export class Scene extends AbstractScene implements IAnimatable, IClipPlanesHold
         if (!options || !options.virtual) {
             this._engine.onNewSceneAddedObservable.notifyObservers(this);
         }
+
+        if ('ComputePressureObserver' in window) {

```

Comment:
maybe having this check as a static on the ComputePressureObserver class?

Reply:
Good idea!